diabetes_regression/.amlignore

```diff
@@ -0,0 +1,3 @@
+# We use yml files to configure deployment,
+# but we are not deploying them to compute
```

Comment:
Please put a better description of what this does here IF there is no link to a doc with what this does.